### DISCUSSION TITLE
Общий шаблон для каналов

### DISCUSSION THREAD

**Discussion by dpetruh** [almkuznetsov/bmstudemo#41] [Show and tell]
Нужно продумать, как конкретно должна выглядеть карточка каждого канала, какие пункты должны указывать и тд. На выходе надо получить все каналы с одинаковым оформлением, что должно упростить поиск информации по каждому каналу

**Comment by dpetruh** [comment]
Шаблон, который мы имеем на данный момент:
1. Краткое описание канала
2. Схема канала
3. Физический принцип
4. критерий защищенности

**Reply by dpetruh** [reply]
в некоторых карточках появляется методика проведения СИ. Она нужна? Мы будем добавлять ее во все карточки?

**Reply by almkuznetsov** [reply]
Методика проведения СИ актуальна не для всех каналов, поэтому это скорее опциональный пункт

**Reply by almkuznetsov** [reply]
В принципе я предлагаю то же самое, только чтобы схема была вверху, а то она чем ниже тем менее красиво смотрится
1. Схема канала
2. Описание канала, на основании которого построена схема, либо просто его краткое описание
3. Физический принцип работы канала
4. Основная характеристика (параметр) канала и критерий защищенности. Объединяю потому что по сути критерий неразрывно связан с характеристикой
5. Методика проведения СИ/Методика оценки защищенности (если существует)
